core/src/main/java/com/klaytn/caver/tx/kct/KIP7Token.java

```diff
@@ -0,0 +1,776 @@
+package com.klaytn.caver.tx.kct;
+
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.crypto.KlayCredentials;
+import com.klaytn.caver.methods.response.KlayLogs;
+import com.klaytn.caver.methods.response.KlayTransactionReceipt;
+import com.klaytn.caver.tx.SmartContract;
+import com.klaytn.caver.tx.manager.TransactionManager;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import org.web3j.abi.FunctionEncoder;
+import org.web3j.abi.TypeReference;
+import org.web3j.abi.datatypes.Address;
+import org.web3j.abi.datatypes.Bool;
+import org.web3j.abi.datatypes.Event;
+import org.web3j.abi.datatypes.Function;
+import org.web3j.abi.datatypes.Type;
+import org.web3j.abi.datatypes.Utf8String;
+import org.web3j.abi.datatypes.generated.Uint256;
+import org.web3j.abi.datatypes.generated.Uint8;
+import org.web3j.protocol.core.RemoteCall;
+import org.web3j.tx.gas.ContractGasProvider;
+
+
+public class KIP7Token extends SmartContract {
```

Comment:
Please add a comment that describes how to generate the `BINARY` below. Refer to https://github.com/klaytn/caver-js/blob/dev/packages/caver-klay/caver-klay-kct/kctHelper.js#L85

Reply:
Okay, I'll add it comment as you suggested.

Reply:
I added comment. Would you check it?